build/commands/lib/util.js

```diff
@@ -167,6 +167,9 @@ const util = {
           "src/chrome/tools/test/reference_build/chrome_linux": "%None%",
           "src/chrome/tools/test/reference_build/chrome_mac": "%None%",
           "src/chrome/tools/test/reference_build/chrome_win": "%None%"
+        },
+        custom_vars: {
+          "checkout_pgo_profiles": "%True%"
```

Comment:
I think we might need to do some research on this because I'm not sure what impact our changes have on the pgo profiles, but we can leave it for now.

Reply:
I'm pretty sure we shouldn't set this, because the impact is kind of upredictable.
It can be workarounded by setting `chrome_pgo_phase=0` as discussed here https://groups.google.com/a/chromium.org/g/chromium-dev/c/-0t4s0RlmOI

Reply:
Turned off for now in d3c599a308dfb4558064208d6266e4573832b208
Follow up: https://github.com/brave/brave-browser/issues/11121